src/qiskit_qec/operators/pauli_list.py

```diff
@@ -160,7 +150,10 @@ def _from_paulis(data: Iterable, input_qubit_order: str = "right-to-left"):
             matrix[index][num_qubits : num_qubits + pauli.num_qubits] = pauli.matrix[0][
                 pauli.num_qubits :
             ]
-            phase_exp[index] = pauli._phase_exp
+            if isinstance(pauli._phase_exp, np.ndarray):
+                phase_exp[index] = pauli._phase_exp[0]
+            else:
+                phase_exp[index] = pauli._phase_exp
```

Comment:
This is not correct. It needs to return a numpy array for the _phase_exponent as Pauli is really a list.